components/memix/actions/random/random.mjs

```diff
@@ -0,0 +1,29 @@
+import { axios } from "@pipedream/platform";
+export default {
+  name: "Caption to Memix URL",
+  description:
+    "Generate a Memix share URL with a random template using the supplied caption",
+  key: "memix_random",
```

Comment:
The key pattern is wrong, it should be: `memix-random`

Reply:
no worries. i've changed it!